source/libs/fetch-dom.ts

```diff
@@ -2,12 +2,12 @@ import domify from 'doma';
 import mem from 'mem';
 
 async function fetchDom(url: string): Promise<DocumentFragment>;
-async function fetchDom<TElement extends Element>(url: string, selector: string): Promise<TElement>;
-async function fetchDom(url: string, selector?: string): Promise<Node> {
+async function fetchDom<TElement extends Element>(url: string, selector: string): Promise<TElement | null>;
+async function fetchDom(url: string, selector?: string): Promise<Node | null> {
```

Comment:
Would you be willing to change `fetchDom` to return `undefined` instead of `null`?

Re: https://github.com/sindresorhus/meta/issues/7

We cannot change the browser APIs, but at least we can improve our own APIs.